src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -256,6 +258,22 @@ public String delete(String id, AuthenticatedUserRequest userReq) {
         .submit();
   }
 
+  /**
+   * Conditionally require sharing privileges when a caller is updating a passport identifier. Such
+   * a modification indirectly affects who can access the underlying data.
+   *
+   * @param patchRequest updates to merge with an existing dataset
+   * @return IAM actions needed to apply the requested patch
+   */
+  public List<IamAction> iamActions(DatasetPatchRequestModel patchRequest) {
```

Comment:
I think giving this a more specific name would be helpful, like `iamPatchDatasetActions`

Reply:
Altered!